frontend/src/components/Map/Map.tsx

```
@@ -0,0 +1,101 @@
+  iconAnchor: [12, 24],
+});
+
+const melbourneGeoData: [number, number] = [-37.8136, 144.9631];
+
+const Map = () => {
+  const [position, setPosition] = useState<{ lat: number; lng: number } | null>(null);
+  const [geoData, setGeoData] = useState<IApiSuburbData | null>(null);
+  const dispatch = useAppDispatch();
+
+  const zoomMap = () => {
+    const map = useMap();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)